apps/freelance/src/pages/ProfileQuestions1/hooks.ts

```diff
@@ -0,0 +1,19 @@
+import { profileQ1Payload } from 'redux/services/types/profileQuestions1.types';
+
+import { IProfileQuestions1 } from './model';
+
+export const onFinishLogic = (values: IProfileQuestions1): profileQ1Payload => {
+  const jsonedEdu = JSON.parse(JSON.stringify(values.education));
+  const jsonedWork = JSON.parse(JSON.stringify(values.work_history_wrapper));
+  const returnedValues = {
+    ...values,
+    education_descr: jsonedEdu.information_about_education,
+    education_from: jsonedEdu.education_from.substring(0, 4),
```

Comment:
it look like something very bad, why you need substring, try to find how do it without substring